src/app/modules/main/module.nim

```diff
@@ -462,13 +462,20 @@ proc connectForNotificationsOnly[T](self: Module[T]) =
 
   self.events.on(SIGNAL_TRANSACTION_SENT) do(e:Args):
     let args = TransactionSentArgs(e)
-    self.view.showToastTransactionSent(args.chainId, args.txHash, args.uuid, args.error)
+    self.view.showToastTransactionSent(args.chainId, args.txHash, args.uuid, args.error,
```

Comment:
ahhh most functions related to transactions have this ever increasing list of parameters, most of which are only necessary in some types of txs... We need to refactor these with structs, I guess it can be done along with the big rework next cycle.